src/canvas-extractor/edfi_canvas_extractor/graphql/extractor.py

```diff
@@ -45,8 +73,8 @@ def get_from_canvas(self, query: str):
         Dict JSON Object
         """
 
-        GRAPHQL_URL = f"{self.base_url}/api/graphql"
-        GRAPHQL_AUTH = {'Authorization': f'Bearer {self.access_token}'}
+        GRAPHQL_URL = f"{self.url}/api/graphql"
+        GRAPHQL_AUTH = {'Authorization': f'Bearer {self.token}'}
 
         try:
             fetch = requests.post(
```

Comment:
I just ran the GraphQL query in Postman, with page size of only 20. I got a gateway 504 error. Then tried again a few seconds later. It completed, but it took 48.67 seconds to run. 

We might need to add a parameter to control the timeout setting on the request. This should probably be a separate ticket though.

Reply:
https://stackoverflow.com/questions/38965086/why-is-default-timeout-for-python-requests-get-different-for-different-machine

I have seen a couple of other articles mentioning this too: there is no default timeout setting. So this will just run until there is completion or a gateway timeout. That's fine with me. Therefore no new ticket is needed at this time, and we will not add a timeout setting to the `request.post` command.